flow/flow_ebos_foam.cpp

```diff
@@ -64,6 +69,13 @@ int flowEbosFoamMain(int argc, char** argv, bool outputCout, bool outputFiles)
 #endif
 
     Opm::FlowMainEbos<TTAG(EclFlowFoamProblem)> mainfunc;
+    Opm::Parser parser;
+    const auto deck = parser.parseFile( argv[1] );
+    auto ecl_state = Opm::EclipseState(deck);
+    auto schedule  = Opm::Schedule(deck, ecl_state);
+    auto summary_config = Opm::SummaryConfig(deck, schedule, ecl_state.getTableManager());
```

Comment:
this would completely kills parallel. see flow.cpp.

Reply:
I had misunderstood something here - removing again.